test/ssl-tests/17-renegotiate.cnf.in

```diff
@@ -261,6 +261,37 @@ our @tests_tls1_2 = (
             "ResumptionExpected" => "No",
             "ExpectedResult" => "ServerFail"
         }
+    },
+    {
+        name => "allow-client-renegotiation",
+        server => {
+            "MaxProtocol" => "TLSv1.2",
+            "Options" => "ClientRenegotiation",
+        },
+        client => {
+            "MaxProtocol" => "TLSv1.2"
+        },
+        test => {
+            "Method" => "TLS",
+            "HandshakeMode" => "RenegotiateClient",
+            "ResumptionExpected" => "No",
```

Comment:
This code:

https://github.com/openssl/openssl/blob/eb1b66f00ca4e1fb6f9e815e8686768b6d81722d/test/helpers/handshake.c#L894-L919

Suggests that the answer here should be "Yes"